java/fundamentals/java-basics/java-identifiers.md

```diff
@@ -27,12 +27,12 @@ inAlgoPool: false
 ---
 ## Content
 
-In Java, every class, method, or variable has to have a name. Formally, this name is called an **identifier**.
+In Java, every class, method, or variable has to have a name. Formally, this name is called a **identifier**.
```

Comment:
identifier starts with an I, which is a vowel, so it should have "an" rather than "a"